### DISCUSSION TITLE
Welcome to motion-ai Discussions!

### DISCUSSION THREAD

**Discussion by dcmartin** [dcmartin/motion-ai#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by calebgab** [comment]
Good evening. Caleb here from New Zealand. Have been running Home Assistant for the past year and a half and Blue Iris for the past year. I'm keen on using Motion-AI but just have a couple questions about the install. I'm running Home Assistant OS in a VM. Can Motion-AI be installed with this setup? Or do you have to do a clean OS install and then install Motion-AI, and use your scripts for installing HomeAssistant on the machine afterwards? The documentation you've done looks quite detailed and awesome but maybe lacking some clear info around how to get it installed / having to start with a fresh Home Assistant install? This is such an amazing project though and super keen on getting it going!

**Reply by dcmartin** [reply]
I never saw this message; I apologize.  The installation instructions are based on a clean install of Ubuntu18.04 on AMD64 and ARM64 (aarch64) machines and Rasbian Buster for Rpi3/4.

You can install on Ubuntu20.04 and Debian10, but YMMV.

To try it out, I recommend a 2 vCPU and 2 GB virtual machine with a fresh install of Ubuntu18.04
